core/src/main/scala/io/qbeast/core/model/QbeastFile.scala

```diff
@@ -9,6 +9,8 @@ package io.qbeast.core.model
  * @param maxWeight
  * @param state
  * @param elementCount
+ * @param size
+ * @param modificationTime
  */
 case class QbeastFile(
```

Comment:
Maybe call it QbeastBlock, or just Block, as we agreed earlier to do not call the single objects/files files.